sdk/provisioning/Azure.Provisioning.AppConfiguration/README.md

```diff
@@ -22,6 +22,52 @@ dotnet add package Azure.Provisioning.AppConfiguration
 
 This library allows you to specify your infrastructure in a declarative style using dotnet.  You can then use azd to deploy your infrastructure to Azure directly without needing to write or maintain bicep or arm templates.
 
+## Examples
+
+### Create a basic AppConfiguration resource
```

Comment:
Could we align the casing in subsection names? All the words should have its first letter capitalized.